brightwind/load/station.py

```diff
@@ -307,6 +309,13 @@ def _raise_child(dictionary, child_to_raise):
                            'slope', 'offset', 'sensitivity',
                            'notes', 'update_at']
     },
+    'logger_measurement_config': {
+        'prefix_separator': '.',
+        'title_prefix': 'Logger ',
+        'keys_to_prefix': ['height_m', 'height_reference_id', 'serial_number',
```

Comment:
Is 'height_reference' part of logger measurement config?

Reply:
solved